src/app/src/util/util.ga.js

```diff
@@ -0,0 +1,27 @@
+export const GA_TRACKING = 'GA_TRACKING';
+export const HAS_ACCEPTED_GA_TRACKING = 'HAS_ACCEPTED_GA_TRACKING';
+export const HAS_REJECTED_GA_TRACKING = 'HAS_REJECTED_GA_TRACKING';
+
+export const userHasAcceptedOrRejectedGATracking = () =>
+    Boolean(localStorage.getItem(GA_TRACKING));
+
+export const userHasAcceptedGATracking = () =>
+    localStorage.getItem(GA_TRACKING) === HAS_ACCEPTED_GA_TRACKING;
+
+export const userHasRejectedGATracking = () =>
+    localStorage.getItem(GA_TRACKING) === HAS_REJECTED_GA_TRACKING;
+
+export const rejectGATracking = () => {
+    localStorage.setItem(GA_TRACKING, HAS_REJECTED_GA_TRACKING);
+};
+
+export const acceptGATracking = () => {
+    // If user has already rejected GA tracking, this is a noop
+    if (!userHasRejectedGATracking()) {
+        localStorage.setItem(GA_TRACKING, HAS_ACCEPTED_GA_TRACKING);
+    }
+};
+
+export const clearGATracking = () => {
```

Comment:
Consider naming this `clearGATrackingDecision` so we are not implying that this actually clears tracking data.

Reply:
Sounds good -- will do!

Reply:
Added in force-pushed amended commit a186a04ddf5e72987abed8678fc4bb8bdeeab13b